File: src/main/java/render/GuiLoadingScreen.java
package render;

import org.joml.Matrix4f;

public class GuiLoadingScreen extends GuiScreen {
    public int chunksLeft = 0;

    public void render(Matrix4f proj) {
        FontRenderer.FONT_RENDERER.renderWithShadowImmediate(proj, 128.0f, 128.0f, "Loading "+ chunksLeft + " chunks...", 1.0f);
    }
}

File: src/main/kotlin/render/GuiLoadingScreen.kt
package render

import org.joml.Matrix4f

class GuiLoadingScreen : GuiScreen() {
    var chunksLeft = 0
    override fun render(proj: Matrix4f) {
        Window.FONT_RENDERER.renderWithShadowImmediate(proj, 128.0f, 128.0f, "Loading "+ chunksLeft + " chunks...", 1.0f)
    }
}